docs/vocabulary.md

```diff
@@ -416,6 +416,27 @@ In the lists and table below, Software Artifacts includes: Documentation Source
 * Inputs: Secrets, Pipeline Environment, Pipeline Workspace, Pipeline Utility Tools
 * Outputs: Stage Results and Return Codes, Logs
 
+#### OSS Introduction Stage
+* Semantics: Download open source software source code in order to analyze it to determine if it adheres to organizational policies from licensing, security, and compliance perspective.
+* Aliases: Introduce
+* Inputs: Public/Upstream URL to source code repository of the primary OSS, version of the primary OSS, Organizational Policies
+* Outputs: Metadata, records, logs, SBOM, evaluation result/score based on organizational policies
+* Other Results and Side Effects: The Introducation stage may also download community generated SBOM for the given version of the OSS if available, generate SBOM using the tools organization uses, generate statistics for determining community health (e.g., no of contributors, release frequency), analyze the primary OSS to identify its dependencies, identify license/copyright information, analyze software for vulnerabilities and malware, perform basic quality analysis. One or more of these outputs may contribute to final evaluation result/score and may result in blocking the introduction of the primary OSS.
+
+#### OSS Storage Stage
+* Semantics: Download the source code of the primary OSS from its origin, analyze it for its dependencies, download the source code of the dependencies from their origin, and store them on internal/secure/trusted repositories for further activities as well as for making it available for consumption by the rest of the development organization.
+* Aliases: Store
```

Comment:
Would this include copying open source packages into an internal Artifactory, for example?  In that case I would add "Mirror" as an alias.

Reply:
I didn't think about packages since this stage deals with storing the source code/repository. But mirror as an alias makes sense so I'll add it to aliases.

Reply:
How about "Vendor" as an alias?
Ref: https://stackoverflow.com/questions/26217488/what-is-vendoring